README.md

```diff
@@ -69,60 +71,98 @@ This section explains how to install the latest stable release of
 Spyder. If you prefer testing the development version, please use
 the `bootstrap` script (see next section).
 
-The easiest way to install Spyder is:
+### The Easy/Recommended Way: Anaconda
 
-### On Windows:
+The easiest way to install Spyder for any of our platform, and
+the way we recommend to avoid unexpected issues we aren't able to
+help you with, is to download it as part of the
+[Anaconda](https://www.anaconda.com/download/) distribution, and keep
+it updated and install other packages with the `conda` package
+and environment manager.
 
-Using one (and only one) of these scientific Python distributions:
+If in doubt, you should install via this method; it generally has the
+least likelihood of potential pitfalls for non-experts, and we may be
+able to provide limited assistance if you do run into trouble.
 
-1. [Anaconda](http://continuum.io/downloads)
-2. [WinPython](https://winpython.github.io/)
-3. [Python(x,y)](http://python-xy.github.io)
+### The Harder Way: Alternative Distributions
 
-### On Mac OSX:
+**Important Note:** While we offer alternative options for users who
+desire them, if you encounter installation issues you are unable to
+solve on your own, we recommend you use Anaconda instead, as we are
+generally unable to offer individual assistance for problems specific
+to installing via these alternative distributions.
 
-- Using the [Anaconda Distribution](http://continuum.io/downloads).
-- Through [MacPorts](http://www.macports.org/).
+**Windows**
 
-### On GNU/Linux
+Spyder is included in these other scientific Python distributions:
 
-- Through your distribution package manager (i.e. `apt-get`, `yum`,
-  etc).
-- Using the [Anaconda Distribution](http://continuum.io/downloads).
-- Installing from source (see below).
+* [WinPython](https://winpython.github.io/)
+* [Python(x,y)](http://python-xy.github.io)
 
-### Cross-platform way from source
+**macOS**
 
-You can also install Spyder with the `pip` package manager, which comes by
-default with most Python installations. For that you need to use the
-command:
+Spyder can be obtained through through
+[MacPorts](http://www.macports.org/).
 
-    pip install spyder
+**GNU/Linux**
 
-To upgrade Spyder to its latest version, if it was installed before, you need
-to run
+* You can often get Spyder through your distribution's package
+manager (i.e. `apt-get`, `yum`, etc), or install from source
+(see below).
 
-    pip install --upgrade spyder
+### The Expert/Cross-platform Way: Installing from source with pip
```

Comment:
Two problems here: i) Anaconda is also cross-platform; ii) pip installs binaries these days. So I think this should be:

> The Experts Way: Installing with pip

Reply:
Agreed; likely whoever originally wrote it did so long ago, and was thinking along the lines that with it being from source, it could be installed on any platform Python could (wider than just Anaconda). Changed to *"The Expert Way: Installing with pip"* (minor grammar tweak to the above).